src/components/common/LoadingSpinner.svelte

```diff
@@ -0,0 +1,28 @@
+<div
```

Comment:
This loader isn't based off any designs, so maybe there's an existing/more appropriate UI element I should be using here?

Reply:
This loader looks good to me!